### PR TITLE
f-registration@v3.0.0 - Use new sass syntax.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Changed
- **breaking changes** Update to `@use` and `@forward` SASS syntax.

### CODE REVIEW

File: packages/components/pages/f-registration/package.json [hunk 1]
```diff
@@ -58,9 +58,10 @@
     "@justeat/browserslist-config-fozzie": ">=1.1.1"
   },
   "devDependencies": {
-    "@justeat/f-button": "3.0.2",
```
Comment (jamieomaguire) [inline packages/components/pages/f-registration/package.json:61]: I spoke to Ben earlier and my understanding is for these we should use `major.x` for now (excluding the fozzie beta)

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/pages/f-registration/package.json (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/pages/f-registration/package.json:61]: Fixed.

**Review by jamieomaguire — APPROVED** [review]

File: packages/components/pages/f-registration/package.json [hunk 2]
```diff
@@ -58,9 +58,10 @@
...
-    "@justeat/f-card": "3.0.0",
-    "@justeat/f-error-message": "1.0.0",
+    "@justeat/fozzie": "9.0.0-beta.6",
+    "@justeat/f-button": "4.x",
```
Comment (zlatinivanov-je) [inline packages/components/pages/f-registration/package.json:62]: I may be missing something here, but what is the logic behind defining the version of the dependencies in such way?

**Review by zlatinivanov-je — APPROVED** [review]
Just a question to clarify the decision on versioning dependencies

File: packages/components/pages/f-registration/package.json [hunk 3]
```diff
@@ -58,9 +58,10 @@
...
+    "@justeat/f-card": "4.x",
+    "@justeat/f-error-message": "2.x",
```
Comment (zlatinivanov-je) [inline packages/components/pages/f-registration/package.json:64]: Don't see the use of this? `@justeat/f-error-message`

File: packages/components/pages/f-registration/package.json (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/pages/f-registration/package.json:62]: There's a PR which explains but in short the devDeps need to match the peerDep so yarn pulls down the correct versions. https://github.com/justeat/fozzie-components/pull/1956

**Pull request merged**